drape_frontend/stylist.cpp

```diff
@@ -229,6 +229,8 @@ void CaptionDescription::FormatCaptions(FeatureType const & f,
     {
       if (m_mainText.empty() || m_houseNumber.find(m_mainText) != string::npos)
         m_houseNumber.swap(m_mainText);
+      else
+        m_mainText += (" (" + m_houseNumber + ")");
```

Comment:
Нет, не голосовать. Если нужно рисовать и имя, и номер, нужно делать это правильно. Например, рисовать номер снизу от названия, отдельным стилем.

> On 12.4.2016, at 19.56, Viktor Govako <notifications@github.com> wrote:
> 
> In drape_frontend/stylist.cpp:
> 
> > @@ -229,6 +229,8 @@ void CaptionDescription::FormatCaptions(FeatureType const & f,
> >      {
> >        if (m_mainText.empty() || m_houseNumber.find(m_mainText) != string::npos)
> >          m_houseNumber.swap(m_mainText);
> > +      else
> > +        m_mainText += (" (" + m_houseNumber + ")");
> Голосовать? :)
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>